apps/web/src/templates/nft-template.tsx

```diff
@@ -35,7 +35,6 @@ function getTransferrableStatus(
 ) {
   if (!currentUserAddress) return 'noUser'
   if (collectible.currentOwnerAddress !== currentUserAddress) return 'notOwner'
-  if (collectible.isFrozen) return 'frozen'
```

Comment:
Since we set `defaultFrozen` to true again, this is no longer preventing us from doing exports. The export/import code is also updated to rely on clawbacks.